src/flows/shared/steps/OrganizeCollection/Editor/CollectionEditor.tsx

```diff
@@ -39,6 +40,14 @@ function CollectionEditor() {
 
   const unassignedNfts = useUnassignedNfts({ skipCache: false });
 
+  const { collections } = useAuthenticatedGallery();
+  const nftsWithinCollectionBeingEdited = useMemo(() => {
+    const collectionBeingEdited = collections.find(
+      (coll) => coll.id === collectionIdBeingEdited
+    );
+    return collectionBeingEdited?.nfts ?? [];
+  }, [collectionIdBeingEdited, collections]);
```

Comment:
this is mostly working, still some weird bug with the context

Reply:
What's the bug? Should we leave a todo to remember

Reply:
fixed!